src/iidm/Network.cpp

```diff
@@ -43,6 +48,15 @@ namespace powsybl {
 
 namespace iidm {
 
+static const converter::Parameter& ANONYMISED_PARAMETER = converter::Parameter(converter::ANONYMISED, converter::ParameterType::BOOLEAN, "Anonymise exported network", false);
+static const converter::Parameter& INDENT_PARAMETER = converter::Parameter(converter::INDENT, converter::ParameterType::BOOLEAN, "Indent export output file", true);
+static const converter::Parameter& EXTENSIONS_LIST_PARAMETER = converter::Parameter(converter::EXTENSIONS_LIST, converter::ParameterType::STRING_LIST, "The list of exported extensions", std::set<std::string>());
+static const converter::Parameter& ONLY_MAIN_CC_PARAMETER = converter::Parameter(converter::ONLY_MAIN_CC, converter::ParameterType::BOOLEAN, "Export only main CC", false);
+static const converter::Parameter& THROW_EXCEPTION_IF_EXTENSION_NOT_FOUND_PARAMETER = converter::Parameter(converter::THROW_EXCEPTION_IF_EXTENSION_NOT_FOUND, converter::ParameterType::BOOLEAN, "Throw exception if extension not found", false);
+static const converter::Parameter& TOPOLOGY_LEVEL_PARAMETER = converter::Parameter(converter::TOPOLOGY_LEVEL, converter::ParameterType::STRING, "Export network in this topology level", std::string("NODE_BREAKER"));
```

Comment:
Not sure you need to explicitly call the conversion from `char*` to `std::string`

Reply:
Yes it is needed, unless there is an implicit conversion into bool, clang-tidy highlights it:
```
/mnt/e/Projet_SRO/rte/powsybl-iidm4cpp/src/iidm/Network.cpp:57:202: error: implicit conversion 'const char *' -> bool [readability-implicit-bool-conversion,-warnings-as-errors]
static const converter::Parameter& TOPOLOGY_LEVEL_PARAMETER = converter::Parameter(converter::xml::TOPOLOGY_LEVEL, converter::Parameter::ParameterType::STRING, "Export network in this topology level", "NODE_BREAKER");
```